### PR TITLE
1101 ls abbreviation on first use

### PR DESCRIPTION
**Pull request by deeppatelcs**
Moved **ls** abbreviation  explanation from episode 2 to episode 1.

### CODE REVIEW

**Comment by slel** [comment]
Please add a comma and a newline after "which stands for 'listing'".

**Comment by gcapes** [comment]
Hi @deeppatelcs,  and thanks for the PR.

The line in ep1 is now a bit long and difficult to read. I would rephrase like this:
> So let's try our first command, `ls` which is short for listing. 
This command will list the contents of the current directory:

**Comment by deeppatelcs** [comment]
Hi @slel and @gcapes I have rephrased it suggested by @gcapes in above comment.
